others/matrix_exponentiation.cpp

```diff
@@ -1,5 +1,5 @@
 /*
-Matrix Exponentiation. If the problem can be solved with DP but constaints are
+Matrix Exponentiation. If the problem can be solved with DP but constalls are
```

Comment:
canstalls

Reply:
Constraints?

Reply:
Yeah. I replaced all the 'int' with 'll'. I'll get this thing.